ipatests/test_integration/test_forced_client_reenrollment.py

```diff
@@ -155,20 +166,22 @@ def uninstall_client(self):
     def restore_client(self):
         client = self.clients[0]
 
-        client.run_command([
-            'iptables',
-            '-A', 'INPUT',
-            '-j', 'ACCEPT',
-            '-p', 'tcp',
-            '--dport', '22'
-        ])
-        client.run_command([
-            'iptables',
-            '-A', 'INPUT',
-            '-j', 'REJECT',
-            '-p', 'all',
-            '--source', self.master.ip
-        ])
+        for host in [self.master] + self.replicas:
+
+            client.run_command([
```

Comment:
We also need to cut connection to all potential replicas as they can tell master the client was uninstalled.

Reply:
My point is that you are removing the same iptable rule (port 22) every time. It is not unique per master.

Reply:
Same issue. The call to iptables to add the rule for port 22 should be moved outside of the loop. Otherwise the rule is potentially added multiple times (I must have misread it earlier and saw it as a delete instead of an add).